index.rst

```diff
@@ -87,6 +87,7 @@ Miscellaneous
    doc/arm_jogging/arm_jogging_tutorial
    doc/benchmarking/benchmarking_tutorial
    doc/tests/tests_tutorial
+   doc/hand_eye_calibration/hand_eye_calibration_tutorial
```

Comment:
I think this should be listed under "Integrate with a New Robot" right beneath  ``doc/perception_pipeline/perception_pipeline_tutorial``

Reply:
Done.